packages/docs/src/routes/docs/integrations/tauri/index.mdx

```diff
@@ -14,3 +14,52 @@ created_at: '2023-05-14T08:01:52Z'
 Tauri is a framework to build desktop applications with any frontend framework and a Rust core.
 
 Learn more on the [Tauri website](https://tauri.app/v1/guides/getting-started/setup/qwik/).
+
+## Adapter
+Make sure you use the [Static Adapter](https://qwik.dev/docs/deployments/static/).
+
+Then you can build your app with 
+
+```shell
+npm run build
+```
+
+## Add Tauri
+
+To install Tauri via npm run:
+
+```shell
+npm install -D @tauri-apps/cli
+```
+
+In your `package.json`, add this script:
+
+```json title="package.json"
+```
```

Comment:
Thanks for your help @nevthereal 
did you miss the script here?

Reply:
oh yes. should I add them real quick?

Reply:
added!